pubg_python/domain/base.py

```diff
@@ -54,7 +54,7 @@ def from_dict(self):
         self.relationships = self._data.pop('relationships', {})
 
     def process_relationships(self):
-        if not self.relationships:
+        if not self.relationships or self.type == 'playerSeason':
```

Comment:
Why?

Reply:
I intended to commit on my repo originally and some leftover code may be left from when I tried to implement a new class for playerSeason type. Will clean it up over the weekend. Hasty pull request.

Reply:
An older version of process_relationships() returns a TypeError when dealing with playerSeason relationships because on the season relationship season['data'] is of type dict not of type list as the other relationships. I didn't need metadata so I just stopped process_relationships on playerSeasons. Turning the dict in a list seems to fix it for this specific endpoint but I haven't tested it for all endpoints.
```python
if isinstance(relationship['data'], dict):
               relationship['data'] = list(relationship.values())
```